index.bs

```diff
@@ -4548,15 +4548,22 @@ and similarly, closing or aborting the <code>writable</code> side will implicitl
 
 <h2 id="conventions" class="no-num">Conventions</h2>
 
-This specification uses algorithm conventions very similar to those of [[!ECMASCRIPT]]. However, it deviates in the
-following ways, mostly for brevity. It is hoped (and vaguely planned) that eventually the conventions of ECMAScript
-itself will evolve in these ways.
+This specification uses algorithm conventions very similar to those of [[!ECMASCRIPT]], whose rules should be used to
+interpret it (apart from the exceptions enumerated below). In particular, the objects specified here should be treated
+as <a href="https://tc39.github.io/ecma262/#sec-ecmascript-standard-built-in-objects">built-in objects</a>. For example,
+their <code>name</code> and <code>length</code> properties are derived as described by that specification, as are the
+default property descriptor values.
+
+We also depart from the [[!ECMASCRIPT]] conventions in the following ways, mostly for brevity. It is hoped (and vaguely
+planned) that the conventions of ECMAScript itself will evolve in these ways.
 
 <ul>
   <li> We use destructuring notation in function and method declarations, and assume that the destructuring assignment
     procedure was performed before the algorithm starts.
   <li> We similarly use the default argument notation <code>= {}</code> in a couple of cases.
```

Comment:
Maybe link to the algorithm? I found the interaction in the TransformStream constructor surprising when I looked at it. ie. writableStrategy and readableStrategy are defaulted to undefined, which is not an acceptable value for destructuring assignment, and then when the {Readable,Writable}Stream constructors are called those undefined values are converted to {} by the default arguments there, making the destructuring assignment work.

Reply:
This one isn't as easy to link to... I think it's buried in https://tc39.github.io/ecma262/#sec-functiondeclarationinstantiation . Maybe I'll just expand on it.